HesoyamHospital/WebApplicationTests/Integration/HospitalSurvey/GetAnswersTests.cs

```diff
@@ -42,13 +42,24 @@ public async void Getting_Answers_Per_Each_Doctor_Status_Code_Tests(int id , Htt
 
             response.StatusCode.ShouldBeEquivalentTo(expectedStatusCode);
         }
+
         [Theory]
-        [MemberData(nameof(Data1))]
-        public async void Mean_Values_Per_Section_Status_Code_Tests(string section, HttpStatusCode expectedStatusCode)
+        [MemberData(nameof(Data2))]
+        public async void Getting_Average_Grade_Per_Each_Doctor_Status_Code_Tests(int id, HttpStatusCode expectedStatusCode)
```

Comment:
Test method names are written in snake case writting style.
First character in the word should be lower case. Otherwise it's good.
Reference: [Snake case](https://en.wikipedia.org/wiki/Snake_case)